src/test/java/com/puppycrawl/tools/checkstyle/internal/utils/XdocGenerator.java

```diff
@@ -68,6 +70,9 @@ public static void generateXdocContent() throws Exception {
             finally {
                 sink.close();
             }
+            final File outputFile = new File(pathToFile.replace(".template", ""));
+            final StandardCopyOption exp = StandardCopyOption.REPLACE_EXISTING;
```

Comment:
```suggestion
            final StandardCopyOption copyOption = StandardCopyOption.REPLACE_EXISTING;
```

Reply:
Step 1: Added syntax error in the illegalcatch: 
![image](https://github.com/checkstyle/checkstyle/assets/60972989/bee81f48-5ac8-4238-8d6f-9bb963aea95f)

Step 2: Git Status:
![image](https://github.com/checkstyle/checkstyle/assets/60972989/2997db9a-8e88-4ecb-8031-a1dddc1d74e4)

Step 3: Failed test case
]
![Screenshot 2023-10-22 at 5 15 32 PM](https://github.com/checkstyle/checkstyle/assets/60972989/8f4b82b5-c1a0-4756-8b32-2c8ddb970f89)

ls in the temporary file 
![image](https://github.com/checkstyle/checkstyle/assets/60972989/2040be51-a0de-4920-b6bd-fdd360c6d1d1) @romani @nrmancuso

Reply:
> copyOption

@nrmancuso Done